openlane/common/toolbox.py

```diff
@@ -202,12 +202,12 @@ def get_macro_views_by_priority(
             formats_so_far.append(format)
         return result
 
-    def get_timing_files(
+    def get_timing_files_categorized(
         self,
         config: Mapping[str, Any],
         timing_corner: Optional[str] = None,
         prioritize_nl: bool = False,
-    ) -> Tuple[str, List[str]]:
+    ) -> Tuple[str, List[Path], List[Path], List[Tuple[str, Path]]]:
```

Comment:
`List[Path]` repeated

Reply:
That's intentional: corner, list of libs, list of netlists, list of instances+spefs

Reply:
Got it. Confused `Tuple` with `Union`